### DISCUSSION TITLE
Lualine: How to set icon_only to false?

### DISCUSSION THREAD

**Discussion by agingorange** [LazyVim/LazyVim#2785] [Q&A] [answered]
I have found this in _examples_:

```lua
  {
    "nvim-lualine/lualine.nvim",
    event = "VeryLazy",
    opts = function(_, opts)
      table.insert(opts.sections.lualine_c, ???)
    end,
  },
```

I just can't figure out if this is even possible and if so, how, or if I should replace the whole configuration?

Lazyvim's config is here: https://www.lazyvim.org/plugins/ui#lualinenvim

**Comment by dpetka2001** [comment, marked as answer]
```lua
  {
    "nvim-lualine/lualine.nvim",
    opts = function(_, opts)
      opts.sections.lualine_c[3] = { "filetype", icon_only = false, separator = "", padding = { left = 1, right = 0 } }
    end,
  },
```

**Reply by agingorange** [reply]
Wow, learned something new today, thanks. I really need to do something about my lacking Lua knowledge. This might be a good trigger.
